app/controllers/hiring_staff/vacancies/job_summary_controller.rb

```diff
@@ -24,11 +24,13 @@ def job_summary_form_params
                                          .merge(completed_step: current_step)
   end
 
-  def redirect_to_next_step_if_save_and_continue
+  def redirect_after_validation_and_update
```

Comment:
Might be better to move this to the vacancy controller

Reply:
That would be nice. Unfortunately, none of the methods called redirect_after_validation_and_update are actually identical, because each controller works differently.

Reply:
I'm making them more or less identical, so I will move this method once this PR is merged 👍

Reply:
Are you happy with the renaming of the method?